composer/algorithms/ema/ema.py

```diff
@@ -84,17 +85,21 @@ class EMA(Algorithm):
     See the :doc:`Method Card </method_cards/ema>` for more details.
 
     Args:
-        half_life (str): The time string specifying the half life for terms in the average. A longer half life means
-            old information is remembered longer, a shorter half life means old information is discared sooner.
-            A half life of ``0`` means no averaging is done, an infinite half life means no update is done. Currently
-            only units of epoch ('ep') and batch ('ba'). Value must be an integer.
+        half_life (str, optional): The time string specifying the half life for terms in the average. A longer half
+            life means old information is remembered longer, a shorter half life means old information is discared
+            sooner. A half life of ``0`` means no averaging is done, an infinite half life means no update is done.
+            Currently only units of epoch ('ep') and batch ('ba'). Value must be an integer. Cannot be used if
```

Comment:
How about "Time must be an integer value in the units specified"? The issue here is that our time abstraction doesn't allow for non-integer values, as in "1.5ep" would not be a valid timestring, so the user needs to be made aware of that. 

As aside, I do think "1.5ep" should be a valid timestring, but currently it is not.

Reply:
Yeah, that works. I thought the statement was redundant with "Currently only units of epoch ('ep') and batch ('ba').", but better to be clear.

Reply:
Ok, I added a default value for `half_life`. We'll accept the annoyance if someone does want to run with `smoothing` and has to remember to turn off `half_life` since presumably that means they know what they're about when it comes to EMA.